code/style.css

```diff
@@ -1,7 +1,214 @@
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+  z-index: -1;
+}
+
+.hero-image-section {
+  margin: 0;
+}
+
+/* Paragraph and text  */
+
+h1 {
+  font-size: 3rem;
+  font-weight: 700;
+  width: 280px;
+  text-align: center;
+  margin-bottom: 150px;
+}
+
+h3 {
+  font-size: 2rem;
+  font-weight: 600;
+  margin-top: 0;
+}
+
+.questions {
+  font-size: 1.25rem;
+  font-weight: 400;
+  margin: 0;
+}
+
+p {
+  font-size: 0.98rem;
+  font-weight: 300;
+  line-height: 1.8rem;
+}
+
+hr {
+  margin: 35px 0;
+}
+
+/* Form */
+
+.form-section {
+  background-image: linear-gradient(135deg, #52e5e7, #130cb7);
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.form {
+  background-color: transparent;
+  padding: 90px 80px;
+}
+
+label,
+.form-headline {
+  display: block;
+  margin: 22px 0 10px 0;
+  font-size: 0.75rem;
+  font-weight: 400;
+}
+
+.uppercase {
+  text-transform: uppercase;
+  letter-spacing: 0.2em;
+}
+
+.input-text-block {
+  border: none;
+  background-color: #fff;
+  width: 94%;
+  padding: 15px 0 15px 15px;
+  font-size: 0.9rem;
+  font-weight: 500;
+}
+
+button {
+  border: solid 3px #fff;
+  background-color: transparent;
+  color: #fff;
+  padding: 15px 45px;
+  font-size: 0.9rem;
+  font-weight: 500;
+}
+
+button:hover {
+  background-color: #fff;
+  color: rgb(88, 88, 88);
+  font-weight: 600;
+}
+
+.inline-label {
+  display: inline;
+  font-size: 0.85rem;
+  font-weight: 400;
+  line-height: 25px;
+  margin-right: 15px;
+}
+
+.sign-up-btn {
+  margin-top: 22px;
+  margin-bottom: 5%;
+}
+
+#newsletter {
+  font-size: 50px;
+}
+
+.pointer {
+  cursor: pointer;
+}
+
+/* Accordion */
+
+.FAQ-section {
+  background-color: #110f41;
+  margin-top: 0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.FAQ {
+  padding: 13vw 13vw;
+  width: 100vw;
+}
+
+.answers {
+  display: none;
+}
+
+.active + .answers {
+  display: block;
+}
+
+.accordion-item-header:hover {
+  cursor: pointer;
+}
+
+.questions::after {
+  content: "\002B"; /* Unicode character for "plus" sign (+) */
+  float: right;
+  font-size: 2rem;
+  font-weight: 300;
+  line-height: 1.4rem;
+}
+
+.minus::after {
+  content: "\2212"; /* Unicode character for "minus" sign (-) */
+}
+
+/* Media Query */
+
+@media (min-width: 0px) and (max-width: 500px) {
+  .form {
+    padding: 18vw 13vw;
+  }
+}
+
+/* Change this */
+
+@media (min-width: 501px) and (max-width: 1025px) {
+  .form {
+    width: 350px;
+    box-shadow: 10px 5px 50px 10px rgba(0, 0, 0, 0.25);
+  }
+  .form-section {
+    padding: 10vw 0;
+  }
+  h1 {
+    font-size: 5.6rem;
+    width: 500px;
+  }
+}
 
-.active + .answer {
-  /* some styling here */
+@media (min-width: 1025px) {
+  .form {
+    width: 350px;
+    box-shadow: 10px 5px 50px 10px rgba(0, 0, 0, 0.25);
+  }
+  .form-section {
+    min-height: 100vh;
+  }
+  .FAQ {
+    width: 60vw;
+    padding: 20.5vh 0;
+  }
+  .FAQ-section {
+    margin-top: 0;
+    min-height: 100vh;
```

Comment:
Have not thought about using vh on other elements that video/img! But it ofc totally makes sense, thanks for that!